src/libhwloc_extensions.jl

```diff
@@ -0,0 +1,17 @@
+module LibHwlocExtensions
+
+using ..LibHwloc: libhwloc, Hwloc_jll
```

Comment:
You don't need `Hwloc_jll` here I think?

Reply:
Correct! Removed!